keep/providers/zabbix_provider/zabbix_provider.py

```diff
@@ -30,12 +30,13 @@ class ZabbixProviderAuthConfig:
     Zabbix authentication configuration.
     """
 
-    zabbix_frontend_url: str = dataclasses.field(
+    zabbix_frontend_url: pydantic.AnyHttpUrl = dataclasses.field(
         metadata={
             "required": True,
             "description": "Zabbix Frontend URL",
             "hint": "https://zabbix.example.com",
             "sensitive": False,
+            "validation": "any_http_url"
```

Comment:
@coderabbitai the reason I gave for the OpenShift provider applies here as well. I believe Zabbix is an open-source project which means it can be ran locally and self-hosted.